receiver/sqlqueryreceiver/metrics.go

```diff
@@ -17,16 +17,21 @@ package sqlqueryreceiver // import "github.com/open-telemetry/opentelemetry-coll
 import (
 	"fmt"
 	"strconv"
+	"time"
 
+	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
-func rowToMetric(row metricRow, cfg MetricCfg, dest pmetric.Metric) error {
+// gauge metrics only need a timestamp, not a
+func rowToMetric(row metricRow, cfg MetricCfg, dest pmetric.Metric, startTs pcommon.Timestamp) error {
```

Comment:
it should receive timestamp. start timestamp should be conditionally calculated based on `cfg`